Mathlib/MeasureTheory/Integral/Prod.lean

```diff
@@ -535,6 +536,30 @@ theorem integral_fun_fst (f : α → E) : ∫ z, f z.1 ∂μ.prod ν = ν.real u
   rw [← integral_prod_swap]
   apply integral_fun_snd
 
+section ContinuousLinearMap
+
+variable {E F G : Type*} [NormedAddCommGroup E] [NormedSpace ℝ E] {mE : MeasurableSpace E}
+  [NormedAddCommGroup F] [NormedSpace ℝ F] {mF : MeasurableSpace F}
+  [NormedAddCommGroup G] [NormedSpace ℝ G] {mG : MeasurableSpace G} [CompleteSpace G]
+  {μ : Measure E} [IsProbabilityMeasure μ] {ν : Measure F} [IsProbabilityMeasure ν]
+
+lemma integral_continuousLinearMap_prod' {L : E × F →L[ℝ] G}
```

Comment:
Could you also add the lemma saying that, under these assumptions, `L` is integrable wrt \mu.prod \nu? Often this will come up as a side condition. Same thing for the next lemma. (Maybe assuming that the measures are finite is enough)